examples/src/main/java/io/dapr/examples/workflows/DemoWorkflow.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.examples.workflows;
+
+import com.microsoft.durabletask.TaskCanceledException;
+import io.dapr.workflows.runtime.Workflow;
+import io.dapr.workflows.runtime.WorkflowContext;
+
+import java.time.Duration;
+
+/**
+ * Implementation of the DemoWorkflow for the server side.
+ */
+public class DemoWorkflow extends Workflow {
+
+  @Override
+  public void run(WorkflowContext ctx) {
+    System.out.println("Hi, my name is " + ctx.getName());
+    System.out.println("Waiting for event: 'myEvent'...");

```

Comment:
The problem with `println` statements in workflow code is that they will be repeated each time the workflow replays. In this case, everything before the external event will be written twice. The poor-man's way of avoiding this is to wrap each `println` in a `if (!ctx.getIsReplaying()) { ... }` block. We may want to think of how/whether we can bake this into the workflow context, and probably integrate it with the common Java logging framework.